Tanasee/Tanasee1.xml

```diff
@@ -35,14 +38,684 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    <desc type="LandGrant">Notice of <persName ref="PRS14148Nob"/> granting land to the community of Kəbrān.</desc>
+                                </item>
+                                <item xml:id="a26">
+                                    <locus target="#236vb"/>
+                                    <desc type="LandGrant">Notice by a similar hand as <ref target="#a25"/> directly above.</desc>
+                                </item>
+                                <item xml:id="a27">
+                                    <locus target="#237ra"/>
+                                    <desc type="LandGrant">Note written by a hand similar to that of the previous <ref target="#a25 #a26"/>, but taller 
+                                        and written with an irregular margin to the right on a land grant in the time of <persName ref="PRS6229LebnaDe"/>.</desc>
+                                </item>
+                                <item xml:id="a28">
+                                    <locus target="#237ra"/>
+                                    <desc type="LandGrant"/>
+                                </item>
+                                <item xml:id="a29">
+                                    <locus target="#237va"/>
+                                    <desc type="LandGrant">Land grant written in very small characters and partly not legible in the microfilm images: 
+                                        <foreign xml:lang="gez">ኍልቈ፡ ምድር፡ ዘወራሚት፡ ቢኖራ፡ ፪ምድር፡ ዘ፩ምድር፡ 
+                                            <gap reason="ellipsis" unit="lines" quantity="19"/></foreign></desc>
+                                </item>
+                                <item xml:id="a30">
+                                    <locus target="#237vb"/>
+                                    <desc type="LandGrant">Land grants issued by a king named <foreign xml:lang="gez">አድያም፡ ሰገድ፡</foreign>, who can be 
+                                        identified as either <persName ref="PRS5616IyasuI"/>, <persName ref="PRS5617IyasuII"/> or 
+                                        <persName ref="PRS5636IyoasI"/>.</desc>
+                                </item>
+                                <item xml:id="a31">
+                                    <locus target="#237vb"/>
+                                    <desc type="RecordDistribution">Possibly a record of taxes to be distributed to a number of local chiefs.</desc>
+                                </item>
+                                <item xml:id="e1">
+                                    <locus target="#2r"/>
+                                    <desc type="OwnershipNote"><foreign xml:lang="gez">ዘደሴት፡ ቅዱስ፡ ገብርኤል፡ መጽሐ።</foreign> by a modern Ethiopian 
+                                        hand on top of the folio with previously erased text.</desc>
+                                </item>
+                                <item xml:id="e2">
```

Comment:
I don't understand according to which criterium the notes are categorized as additions or extras :) There are several inventories among the extras which, only by reading the description, I would classify as additions.

Reply:
*I meant: "I think, there is no reason (*not) to move e5 to the additions. It must become a20 then to keep the order in the codex." Sorry for the mistake! I categorized all extras as additions now, except for the general note on single word corrections (e1) and an unclear note on f. 232ra (e2).